.gitignore

```diff
@@ -1,4 +0,0 @@
-_site/
-.sass-cache/
-.jekyll-metadata
-*~
```

Comment:
Style-guide is its own branch now. That's why it was deleted here. I have 4 branches now: master, style-guide, radioactive-decay, atomic-structure. 

It looks like I made quite a mess. I'm not sure how I fix that then on my own in the right way. If radioactive-decay is deleted here, I could just make a new pull request for that branch again if that's the easiest route. 

I'd like to understand how to do this work correctly because I'm going to apply this to my pyroprocessing code development, and it's important not to screw up again here if we're going to be adding lots of nodes.